### PR TITLE
TF-381 Apply new UI for browser list emails

### PR DESCRIPTION
**Pull request by dab246**
https://user-images.githubusercontent.com/80730648/159432933-90f7be78-e6a2-467e-a271-8913c00af2fe.mov

### CODE REVIEW

File: lib/features/caching/email_cache_client.dart [hunk 1]
```diff
@@ -14,8 +14,9 @@ class EmailCacheClient extends HiveCacheClient<EmailCache> {
     return Future.sync(() async {
       if (Hive.isBoxOpen(tableName)) {
         return Hive.box<EmailCache>(tableName);
+      } else {
+        return Hive.openBox<EmailCache>(tableName);
```
Comment (hoangdat) [inline lib/features/caching/email_cache_client.dart:18]: What happens with this, I see both are similar?

File: lib/features/session/presentation/session_controller.dart [hunk 2]
```diff
@@ -40,7 +41,7 @@ class SessionController extends GetxController {
 
   void _goToLogin() {
     _deleteCredential();
-    _clearAllCache();
+    if (!kIsWeb) _clearAllCache();
```
Comment (hoangdat) [inline lib/features/session/presentation/session_controller.dart:44]: Why not clear the cache on the web?

Comment (hoangdat) [reply lib/features/session/presentation/session_controller.dart:44]: it only occurs in web?

Comment (dab246) [reply lib/features/session/presentation/session_controller.dart:44]: Follow [https://github.com/hivedb/hive/issues/896](https://github.com/hivedb/hive/issues/896)

**Review by hoangdat — APPROVED** [review]

**Pull request merged**